### ISSUE TITLE
Issue of SideMenu on Safari

### ISSUE THREAD

**Issue by jsfan3** [codenameone/CodenameOne#2659]
On the Javascript port, the SideMenu works correctly, but there is an issue in Safari only (not in Chrome). The problem is that Safari uses the swipe to go back or forward in the history, while the SideMenu uses the same swipe to open or close itself. This creates a conflict and weird effects.

Is it possible to disable the Safari default behavior about the swipe? If it's not possible, is it possible to disable the SideMenu swipe only on Safari (leaving it in the native apps and in other browsers)?

**Comment by jsfan3** [comment]
The same issue is with Firefox on iPhone. I didn’t test yet with Firefox on Android.

**Comment by shannah** [comment]
My initial googling is making the situation look bleak.
https://stackoverflow.com/questions/18889666/ios-7-is-there-a-way-to-disable-the-swipe-back-and-forward-functionality-in-sa

TLDR:  There is no way to disable this swipe forward/back behaviour in Safari.  The workarounds suggested in this thread and other places is to hack away at the browser history - but this would destroy the normal forward/back behaviour as well so it is not a good option.

**Comment by jsfan3** [comment]
So I suppose that this is the maximum that we can do:
```
        // Default for the side menu
        if (!Display.getInstance().isSimulator() && "HTML5".equals(CN.getPlatformName())) {
            Toolbar.setEnableSideMenuSwipe(false);
        } else {
            Toolbar.setEnableSideMenuSwipe(true);
        }
```

**Comment by shannah** [comment]
I'm going to close this issue for now with the wontfix label.  If something changes and it looks like it can be fixed, I'll reopen it.
